src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Triggers/EventHubTriggerInput.cs

```diff
@@ -51,6 +52,35 @@ public EventHubTriggerInput GetSingleEventTriggerInput(int idx)
         public EventData GetSingleEventData()
         {
             return this.Events[this._selector];
-        }        
+        }
+
+        public Dictionary<string, string> GetTriggerDetails(PartitionContext context)
+        {
+            string offset, enqueueTimeUtc, sequenceNumber;
+            if (IsSingleDispatch)
+            {
+                offset = Events[0].SystemProperties?.Offset;
+                enqueueTimeUtc = Events[0].SystemProperties?.EnqueuedTimeUtc.ToString("o");
+                sequenceNumber = Events[0].SystemProperties?.SequenceNumber.ToString();
+            }
+            else
+            {
+                EventData first = Events[0];
+                EventData last = Events[Events.Length - 1];
+
+                offset = $"{first.SystemProperties?.Offset}-{last.SystemProperties?.Offset}";
+                enqueueTimeUtc = $"{first.SystemProperties?.EnqueuedTimeUtc.ToString("o")}-{last.SystemProperties?.EnqueuedTimeUtc.ToString("o")}";
+                sequenceNumber = $"{first.SystemProperties?.SequenceNumber}-{last.SystemProperties?.SequenceNumber}";
+            }
+
+            return new Dictionary<string, string>()
+            {
+                { "PartionId", context.PartitionId },
+                { "Offset", offset },
+                { "EnqueueTimeUtc", enqueueTimeUtc },
+                { "SequenceNumber", sequenceNumber },
+                { "Count", (Events.Length - 1).ToString()}
```

Comment:
This shouldn't have `- 1` should it? If there's 1 event, we want to write out `1`.

Reply:
yes, I'll fix that